package.json

```diff
@@ -60,6 +58,7 @@
     "@testing-library/jest-dom": "^5.10.1",
     "@testing-library/react": "^10.2.1",
     "@testing-library/user-event": "^12.0.2",
+    "@types/history": "^4.7.6",
```

Comment:
Because @types/react-router has an unpinned version of this it grabs the newest version which doesn't have the types because newer versions of history have them built in. Pinning this prevents the typescript compile/build from failing